varats/plots/paper_config_overview.py

```diff
@@ -0,0 +1,153 @@
+            success = rev in processed_revisions
+            revisions[commit_date.year].append((rev, success))
+
+        projects[case_study.project_name] = revisions
+
+    min_years = []
+    max_years = []
+    for _, revisions in projects.items():
+        years = revisions.keys()
+        min_years.append(min(years))
+        max_years.append(max(years))
+
+    year_range = list(range(min(min_years), max(max_years) + 1))
+    project_names = list(projects.keys())
+
+    result = dict()
+    result['year_range'] = year_range
+    result['project_names'] = project_names
+
+    revs_successful = []
+    revs_total = []
+
+    for _, revisions in projects.items():
+        revs_successful_per_year = []
+        revs_total_per_year = []
+        for year in year_range:
+            revs_in_year = revisions[year]
+            if not revs_in_year:
+                num_revs = np.nan
+                num_successful_revs = np.nan
+            else:
+                num_revs = len(revs_in_year)
+                num_successful_revs = sum(1 for (rev, success) in revs_in_year if success)
+
+            revs_successful_per_year.append(num_successful_revs)
+            revs_total_per_year.append(num_revs)
+
+        revs_successful.append(revs_successful_per_year)
+        revs_total.append(revs_total_per_year)
+
+    result['revs_successful'] = revs_successful
+    result['revs_total'] = revs_total
+
+    return result
+
+
+def _plot_overview_graph(results) -> None:
+    """
+    Create a plot that shows an overview of all case-studies of a paper-config
+    about how many revisions are successful per project/year.
+    """
+    revs_successful = np.asarray(results['revs_successful'])
+    revs_total = np.asarray(results['revs_total'])
+    revs_success_ratio = revs_successful / revs_total
+
+    year_range = results['year_range']
+    project_names = results['project_names']
+
+    labels = (np.asarray(["{0:1.0f}/{1:1.0f}".format(revs_successful, revs_total)
+                          for revs_successful, revs_total
+                          in zip(revs_successful.flatten(), revs_total.flatten())])).reshape(
+                              len(project_names), len(year_range))
+
+    sb.heatmap(revs_success_ratio, annot=labels, fmt='', cmap="BrBG",
+               xticklabels=year_range, yticklabels=project_names,
+               vmin=0, vmax=1,
+               cbar_kws={'label': 'success ratio'})
+
+
+class PaperConfigOverviewPlot(Plot):
+    """
+    Plot showing an overview of all case-studies.
+    """
+
+    @check_required_args(["result_folder"])
+    def __init__(self, **kwargs):
+        super(PaperConfigOverviewPlot, self).__init__("paper_config_overview_plot")
+        self.__saved_extra_args = kwargs
+
+    def plot(self, view_mode):
+        style.use(self.style)
+        _plot_overview_graph(
+            _gen_overview_plot())
+
+    def show(self):
+        self.plot(False)
```

Comment:
```suggestion
        self.plot(True)
```
currently plot does not use this information but, show is called in view mode so we should at least pass the correct value

Reply:
Makes sense